clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -282,4 +367,110 @@ describe("Fides-js TCF", () => {
       });
     });
   });
+
+  describe("cmp api", () => {
+    beforeEach(() => {
+      cy.window().then((win) => {
+        win.__tcfapi("addEventListener", 2, cy.stub().as("TCFEvent"));
+      });
```

Comment:
@eastandwestwind I thought this pattern worked reasonably well for testing the CMP API stuff, might be able to reuse for https://github.com/ethyca/fides/issues/3999

Reply:
oh awesome! I'm so glad to see this worked here, and it'll make my life so much easier 🌟